frontend/src/pages/FunctionalComponent/RecurringRunDetailsV2FC.tsx

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright 2023 The Kubeflow Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { useQuery } from 'react-query';
+import Buttons, { ButtonKeys } from 'src/lib/Buttons';
+import DetailsTable from 'src/components/DetailsTable';
+import { V2beta1Experiment } from 'src/apisv2beta1/experiment';
+import { V2beta1RecurringRun, V2beta1RecurringRunStatus } from 'src/apisv2beta1/recurringrun';
+import { Apis } from 'src/lib/Apis';
+import { PageProps } from 'src/pages/Page';
+import { RoutePage, RouteParams } from 'src/components/Router';
+import { Breadcrumb, ToolbarProps } from 'src/components/Toolbar';
+import { classes } from 'typestyle';
+import { commonCss, padding } from 'src/Css';
+import { KeyValue } from 'src/lib/StaticGraphParser';
+import { formatDateString, enabledDisplayStringV2, errorToMessage } from 'src/lib/Utils';
+import { triggerDisplayString } from 'src/lib/TriggerUtils';
+
+export function RecurringRunDetailsV2FC(props: PageProps) {
+  const { updateBanner, updateToolbar } = props;
+  const [refresh, setRefresh] = useState(true);
+  const [toolbarState, setToolbarState] = useState<ToolbarProps>();
+  const recurringRunId = props.match.params[RouteParams.recurringRunId];
+  const Refresh = () => setRefresh(refreshed => !refreshed);
+
+  const { data: recurringRun, refetch: refetchRecurringRun } = useQuery<
+    V2beta1RecurringRun | undefined,
+    Error
+  >(
+    ['recurringRun'],
```

Comment:
Can we add recurringRunId to the array of query key here as well?

Reference: https://tanstack.com/query/v4/docs/react/guides/query-keys